src/kadmin/ktutil/ktutil.c

```diff
@@ -149,6 +150,7 @@ void ktutil_add_entry(argc, argv)
         }
         if ((strlen(argv[i]) == 2) && !strncmp(argv[i], "-k", 2)) {
             kvno = (krb5_kvno) atoi(argv[++i]);
+            use_kvno++;
```

Comment:
I started work on the CentOS 6.8 krb5 SRPM and then backported that code from there to this git tree. That change is include in a patch called krb5-kvno-230379.patch, for which the the top google reference is https://build.opensuse.org/package/view_file/home:garloff:OTC:curl/krb5-mini/krb5-kvno-230379.patch?expand=1. Since that patch and my change had conflict I did not make a good job of leaving that change alone. If that change is not desired I'll submit a new version without that change. Good catch @apatil.